### PR TITLE
Actually submitting a pull request this time... sugo di pomodoro baby

### PR DESCRIPTION
**Pull request by lukekivi**

### CODE REVIEW

File: Lucas/Italian/sugo_di_pomodoro-marinara.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Sugo di Pomodoro
+a.k.a. Marinara. Make alot and can it. Or not you lazy bastard.
```
Comment (AndrewHaisting) [inline Lucas/Italian/sugo_di_pomodoro-marinara.md:2]: 😂 love this

File: Lucas/Italian/sugo_di_pomodoro-marinara.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Ingrediants:
```
Comment (AndrewHaisting) [inline Lucas/Italian/sugo_di_pomodoro-marinara.md:4]: Typo (maybe its intentional)??

File: Lucas/Italian/sugo_di_pomodoro-marinara.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- **28 oz** Canned whole San Marzano Tomatoes (ideally Certified D.O.P)
+- **1/4 C** Extra virgin olive oil
+- **7 ea** Garlic cloves, sliced thin
+- **Lrg sprig** Basil
+- **TT** Hot stuff - fresno peppers, chili flake, etc.
```
Comment (AndrewHaisting) [inline Lucas/Italian/sugo_di_pomodoro-marinara.md:9]: 👀 I like the TT, gonna start using that

File: Lucas/Italian/sugo_di_pomodoro-marinara.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- **TT** Salt
+- **TT** Balsamic vinegar reduction (Optional)
+  
+## Instructions:
+1. IN a pot brown the garlic in oil. Just until it is golden. Do not let parts of the garlic become **dark** brown or black.
```
Comment (AndrewHaisting) [inline Lucas/Italian/sugo_di_pomodoro-marinara.md:14]:
```suggestion
1. In a pot brown the garlic in oil. Just until it is golden. Do not let parts of the garlic become **dark** brown or black.
```

File: Lucas/Italian/sugo_di_pomodoro-marinara.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+2. Add tomatoes. 
+3. Add hot stuff. I prefer fresnos, which I just smash once or slice in half. You just need to allow liquid to penetrate the skin since you will not cook for a long time.
+4. Add your sprig of basil to the top and allow it to wilt in to the sauce.
+5. Stew at a medium to medium-low heat until the oil is orange in color.
+6. Remove larges stems. Blend until you reach desired chunkiness. I like smooth with a few chunks.
+7. (optional) Return to pot. As far as seasoning goes you are just building a base. No need to season to final levels, this is done in pan before serving. Balsamic vinegar reduction is **NOT** traditional but I like to add a little. Maybe 3 dashes. It brings depth and a sweetness that begins to round out the flavor profile of the sauce.
+8. (optional) CAN SOME! Flavors develop over time so even if you do not can, flavor peaks a few days after cooking.
```
Comment (AndrewHaisting) [inline Lucas/Italian/sugo_di_pomodoro-marinara.md:21]: One small tip here, you can use `1.` for all your numbers and the receiving application will figure it out. Just makes it easier if you add or remove steps you don't have to change every damn number.

**Review by AndrewHaisting — CHANGES_REQUESTED** [review]
Welcome to your first code review :)

File: Lucas/Italian/sugo_di_pomodoro-marinara.md (hunk 5 above)
Comment (lukekivi) [reply Lucas/Italian/sugo_di_pomodoro-marinara.md]: Interesting. VS code automatically increments for me.

**Review by AndrewHaisting — APPROVED** [review]
Looks great! To keep the commit history clean, I would recommend one of two things:

1) using your terminal squash this branch into one commit, then force push to this branch, then merge
2) on the bottom of the pull request, change "Merge pull request" to be "Squash and merge", and do it through the web 

Both have the same outcome.

**Pull request merged**